common-library/src/main/java/com/sdevelopment/skt/common/domain/Product.java

```diff
@@ -0,0 +1,57 @@
+package com.sdevelopment.skt.common.domain;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "products")
+//@Data
+@NamedStoredProcedureQueries({
+        @NamedStoredProcedureQuery(name = "saveProduct",
+                procedureName = "insertproductsp",
+                parameters = {
+                        @StoredProcedureParameter(mode = ParameterMode.IN, name = "_name", type = String.class)
+                }),
+
+        @NamedStoredProcedureQuery(name = "getAllProducts",
+                procedureName = "listproductssp",
+                resultClasses = Product.class)
+})
+public class Product implements Serializable {
+    @Id
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "name")
+    //@NotEmpty(message = "Name may not be empty")
+    @NotBlank(message = "Name may not be empty")
+    private String name;
+
+    public Product() {
+
+    }
+
+    public Product(Integer id, String name) {
+        this.id = id;
+        this.name = name;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+}
```

Comment:
This is intended to be a value class, for this kind of classes is always a good practice to override `equals` and `hashCode` methods.

For more information, check items 8 and 9 of the Effective Java book

Reply:
Done! It's good to cover those topics. I needed to refresh them.